### DISCUSSION TITLE
UTM-SE build dependencies

### DISCUSSION THREAD

**Discussion by jnawaz** [utmapp/UTM#5103] [UTM Help (iOS)] [answered]
I'm running the following command 

`./scripts/build_dependencies.sh -p ios-tci -a arm64` for iOS development however, my mac mini (16GB (RAM) and 500GB storage) runs out of RAM memory and fails to download all the dependencies, rendering my machine useless.

Has anybody else encountered this and / or managed to find a workaround to successfully run the `build_dependencies` and `build_utm` scripts for the UTM-SE target?

**Comment by osy** [comment, marked as answer]
Oh yeah you need to do NCPU=1 or have a machine with a lot more ram.

**Comment by jnawaz** [comment]
I can confirm the NCPU=1 prefix to the build scripts helped and no memory issues when downloading the dependencies on the machine mentioned in the original question. Thank you.
